Getting Started/WalkthroughFirstWPFApp/csharp/ExpenseItIntro.csproj

```diff
@@ -57,38 +21,6 @@
     <Reference Include="PresentationCore" />
```

Comment:
You don't need reference includes here, the SDK + UseWPF adds all the needed references.

Reply:
And in the case where the target framework is .NET Framework 4.X, https://github.com/microsoft/WPF-Samples/blob/master/Directory.Build.props takes care of references.

Reply:
removed.